src/styles/blocks/page.scss

```diff
@@ -0,0 +1,28 @@
+.page {
+  font-family: "Cinzel", serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 27px;
```

Comment:
Consider using a variable for the line-height to make it reusable and easier to maintain.